.github/workflows/build-test.yml

```
@@ -33,6 +43,11 @@ jobs:
       run: |
         pip install .[docs,tests]
         pip install black=="22.6.0" isort=="5.10.1" mypy=="0.981"
+        pip uninstall --yes scikit-learn
+        if [ ${{ matrix.sklearn_version }} == "nightly" ];
+          then pip install --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn;
+          else pip install "scikit-learn>=${{ matrix.sklearn_version }}";
```

shouldn't this be

```suggestion
          else pip install "scikit-learn=${{ matrix.sklearn_version }}";
```

?

---

Right, then maybe this is better?

```shell
$ pip install scikit-learn~=1.0.0
Collecting scikit-learn~=1.0.0
  Downloading scikit_learn-1.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (26.5 MB)
```

https://peps.python.org/pep-0440/#compatible-release